### DISCUSSION TITLE
Elemental value output in postprocessor

### DISCUSSION THREAD

**Discussion by skbarnwal** [idaholab/moose#20495] [Q&A General] [answered]
Hi, 
 I want all elemental output values for every time step in the postprocessor. Is there any option in moose postprocessor? Like if I have 20 elements in FEM and 20 steps then the total output will be 400.

**Comment by GiudGiud** [comment, marked as answer]
Hello

See if this one works for you:
https://mooseframework.inl.gov/source/vectorpostprocessors/ElementValueSampler.html

Guillaume
